apps/ejabberd/src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -60,6 +60,13 @@ worker_prefix() ->
 worker_count(_Host) ->
```

Comment:
It's not good, then, because a different number of workers might be started by `start_workers` than `stop_workers` will try to stop later. Maybe we can just define the default pool size as a macro (instead of a variable in `init`) and use `gen_mod:get_opt` inside `worker_count` each time it is called?